0800-appendix-198116-waterfall.Rmd

```diff
@@ -41,39 +39,32 @@ of Transportation and Infrastructure.
-on two crossings located on tributaries to Waterfall Creek.
+Waterfall Creek is a primary tributary to Station Creek. Wilson and Rabnett (2007) reported that the crossing structure located on Highway 16 and Station Creek (PSCIS ID 124420) has been the subject of numerous assessments and designs with respect to the rehabilitation of fish passage. They ranked this crossing as having the highest priority for rehabilitation in the Bulkley River watershed. According to the Gitxsan Watershed Authority, Station Creek's mouth (Xsi Gwin Sagiiblax) was the location of a fishing site called Xsan Xsagiibil [@wilson_rabnett2007FishPassage]. @rabnett_williams2004Highway16 identified this area as having a high priority for additional assessments. To support rehabilitation efforts, @skrconsultantsltd_2006FishPassage carried out a thorough inspection, provided rehabilitation design options, and identified the natural limits of potential fish distribution. 
 
 <br>
 
-Habitat assessments were conducted on a tributary to Waterfall Creek by
-@irvine2022BulkleyRiver in 2021. Located upstream of PSCIS Crossing
-198116, this tributary flows through New Hazelton and crosses the
-highway. The highway crossing and a crossing further downstream were
+Habitat assessments were conducted downstream of the railway on Waterfall and Station Creek by
+@irvine2022BulkleyRiver in 2021. Waterfall Creek flows through New Hazelton and crosses the railway downstream of PSCIS crossing 198117, then flowing under the highway. The highway crossing and a crossing further downstream were
 assessed and both were deemed to be barriers at the time of survey,
-according to provincial protocols. A detailed background and habitat
+according to provincial protocols. PSCIS crossing 124420 on Station Creek was re-assessed and ranked as a high priority for proceeding to design for replacement. A detailed background and habitat
 assessment can be found in the 2021 report
 [here](https://www.newgraphenvironment.com/fish_passage_skeena_2021_reporting/station-creek---124420-124421-124422---appendix.html).
 
 <br>
 
+Every year the Chicago Creek Community Environmental Enhancement Society carries out a trap and truck operation where coho are moved upstream of Highway 16 on Waterfall Creek from a fence erected up near the junction with the Bulkley River. Houlden, Houlden, and Donas (2001) reported that the incorrect installation of PSCIS crossing 124420 lead to coho stocks being wiped out in the system in the late 1970s. In order to return coho to the stream, the Waterfall Creek Stream Rehabilitation Project was started in 1990. Surplus stocks were moved from Toboggan Creek near Smithers. To help with the mission, the Hazelton Elementary Streamkeepers programme was established in 1993, and the Chicago Creek Community Environmental Enhancement Society was established in 1995 to build a hatchery close by. The hatchery is now shut down [@houlden_donas2002ChicagoCreek]. A restoration
```

Comment:
should be a citation key vs. "Houlden, Houlden, and Donas (2001)" so that it changes if we change the bibtex entry